library/core/functions.general.php

```diff
@@ -3737,9 +3737,10 @@ function setAppCookie($Name, $Value, $Expire = 0, $Force = false) {
      * @param string $String The string to slice.
      * @param int $MaxLength The maximum length of the string.
      * @param string $Suffix The suffix if the string must be sliced mid-sentence.
+     * @param int $MinLength The intended minimum length of the string (slice on sentence if paragraph is too short)
      * @return string
      */
-    function sliceParagraph($String, $MaxLength = 500, $Suffix = '…') {
+    function sliceParagraph($String, $MaxLength = 500, $Suffix = '…', $MinLength = 50) {
```

Comment:
It bothers me to throw a parameter on the end like this. Now param 4 is related to param 2 with something you probably never want to override in between.

What if we made the second parameter default to [32,500] as `list($min, $max)` or defaults to just $max if an int is passed?

I also note your more useful suggestion is still only 27 characters. :) Maybe a lower min default of 32?